sbt

```diff
@@ -573,7 +573,7 @@ Usage: `basename "$0"` [options]
   --sbt-version  <version>   use the specified version of sbt
   --sbt-jar      <path>      use the specified jar as the sbt launcher
 
-  # java version (default: java from PATH, currently $(java -version 2>&1 | grep version))
+  # JAVA_HOME (this will override default JAVA_HOME) 
```

Comment:
I guess we could just remove the line completely?

Reply:
Good idea. I will update the commit to remove it.